practitioner-service/src/main/java/uk/ac/ox/ndph/arts/practitioner_service/repository/IFhirRepository.java

```diff
@@ -0,0 +1,19 @@
+package uk.ac.ox.ndph.arts.practitioner_service.repository;
+
+import org.hl7.fhir.r4.model.Practitioner;
+import uk.ac.ox.ndph.arts.practitioner_service.exception.HttpStatusException;
+
+/**
+* Interface for a FHIR entity repository
+*/
+public interface IFhirRepository {
```

Comment:
In Java you usually do use the **I** interface notation (guessing - since the most common IDEs like Eclipse and IntelliJ have an icon for it).
This applies to all interfaces in this PR.